test/unit/components/termsoverlay.test.js

```diff
@@ -29,4 +30,186 @@ describe('TermsOverlay', function () {
+
+        var termsDetails = iframes[0];
+        expect(termsDetails.props.src).to.equal('https://tidepool.org/terms-of-use/');
+        var privacyDetails = iframes[1];
+        expect(privacyDetails.props.src).to.equal('https://tidepool.org/privacy-policy/');
+
+        expect(termsElem.state.agreed).to.equal(false);
+      });
+    });
+    describe('flow for between 13 and 17 years old', function() {
+      it('shows TOU and PP and asks for parental consent also', function() {
+
+        //Select between 13 and 17
+        var thirteenToSeventeenOpt = TestUtils.scryRenderedDOMComponentsWithTag(termsElem,'input')[1];
+        TestUtils.Simulate.change(thirteenToSeventeenOpt);
+        //Select Continue
+        var ageBtn = TestUtils.findRenderedDOMComponentWithTag(termsElem, 'button');
+        TestUtils.Simulate.click(ageBtn);
+
+        //Check state
+        expect(termsElem.state.ageConfirmed).to.equal(true);
+        expect(termsElem.state.ageSelected).to.equal(termsElem.props.ages.WITH_CONSENT.value);
+        expect(termsElem.state.agreed).to.equal(false);
+        expect(termsElem.state.agreedOnBehalf).to.equal(false);
+
+        var checkboxes = TestUtils.scryRenderedDOMComponentsWithTag(termsElem,'input');
+
+
+        expect(checkboxes.length).to.equal(2);
+        var agreed = checkboxes[0];
+        var agreedOnBehalf = checkboxes[1];
+
+        TestUtils.Simulate.change(agreedOnBehalf);
+        TestUtils.Simulate.change(agreed);
+
+        expect(termsElem.state.agreed).to.equal(true);
+        expect(termsElem.state.agreedOnBehalf).to.equal(true);
+
+        //now we should be able to click the button
+        var buttons = TestUtils.scryRenderedDOMComponentsWithTag(termsElem, 'button');
+        expect(buttons[1].props.children).to.equal('Continue');
+        expect(buttons[1].props.disabled).to.equal(false);
+        expect(buttons[0].props.children).to.equal('Back');
+      });
+      it('will not allow TOU and PP confirmation if both checkboxes are not selected', function() {
+        //Select between 13 and 17
+        var thirteenToSeventeenOpt = TestUtils.scryRenderedDOMComponentsWithTag(termsElem,'input')[1];
+        TestUtils.Simulate.change(thirteenToSeventeenOpt);
+        //Select Continue
+        var ageBtn = TestUtils.findRenderedDOMComponentWithTag(termsElem, 'button');
+        TestUtils.Simulate.click(ageBtn);
+        //age confirmation is now true
+        expect(termsElem.state.ageConfirmed).to.equal(true);
+        expect(termsElem.state.ageSelected).to.equal(termsElem.props.ages.WITH_CONSENT.value);
+        expect(termsElem.state.agreed).to.equal(false);
+        expect(termsElem.state.agreedOnBehalf).to.equal(false);
+
+        var checkboxes = TestUtils.scryRenderedDOMComponentsWithTag(termsElem,'input');
+
+        expect(checkboxes.length).to.equal(2);
+
+        var agreed = checkboxes[0];
+        var agreedOnBehalf = checkboxes[1];
+        //only check one
```

Comment:
Maybe a third test parallel to this one where you only check one of two boxes, but you check the other one?